### PR TITLE
[v1 API Docs] add */triples pages

### PR DESCRIPTION
**Pull request by juliawu**
This PR adds the following v1 REST API documentation pages:
* /v1/triples
* /v1/bulk/triples

### CODE REVIEW

File: api/rest/v1/bulk_triples.md [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+parent: v1 REST
+grand_parent: API
+nav_exclude: true
+published: false
+permalink: /api/rest/v1/bulk/triples
+---
+
+# /v1/bulk/triples
+
+Get [triples](/glossary.html#triple) for multiple entities.
```
Comment (shifucun) [inline api/rest/v1/bulk_triples.md:13]: please change "entities" to "nodes" throughout

File: api/rest/v1/bulk_triples.md [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+
+```json
+{
+  "data":
+  [
+    {
+      "entity": "entity_dcid_1",
+      "triples":
+      {
+        "property_of_entity":
```
Comment (shifucun) [inline api/rest/v1/bulk_triples.md:86]: maybe use "property_1" and "property_2"

**Review by shifucun — COMMENTED** [review]
Overall looks good! Just need to add pagination detail in follow up PR.

File: api/rest/v1/bulk_triples.md (hunk 1 above)
Comment (juliawu) [reply api/rest/v1/bulk_triples.md]: Done!

File: api/rest/v1/bulk_triples.md (hunk 2 above)
Comment (juliawu) [reply api/rest/v1/bulk_triples.md]: Done!

**Review by shifucun — APPROVED** [review]

**Pull request merged**